public/pages/Dashboard/utils/tableUtils.js

```diff
@@ -238,7 +235,7 @@ export const associatedAlertsTableColumns = [
     render: ({ monitor_id, monitor_name }) => {
       return (
         <EuiToolTip content={monitor_name}>
-          <EuiLink href={`${PLUGIN_NAME}#/monitors/${monitor_id}?type='monitor'`} target="_blank">
```

Comment:
why are we removing this constant? just curious

Reply:
Because with the new side nav design, we are registering each page as a separate app with its own base path, so we can no longer rely on PLUGIN_NAME to be the base path at all times. Instead we should just change the hash, since that is that we anyways use to switch the Routes in Main.js

Reply:
For e.g https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/public/plugin.tsx#L101
here we are register Monitors under `/monitors` base path which used to be `/alerting`